interface/patient_file/encounter/forms.php

```diff
@@ -274,7 +274,7 @@
         $(".css_button").hide();
         $(".css_button_small").hide();
         $(".encounter-summary-column:first").show();
-        $(".title:first").text("<?php echo xls("Review"); ?> " + $(".title:first").text() + " ("+<?php echo addslashes($encounter); ?>+")");
+        $(".title:first").text("<?php echo xls("Review"); ?> " + $(".title:first").text() + " ( <?php echo addslashes($encounter); ?> )");
```

Comment:
I will revert the changed line above (this was modified by Jerry a couple days ago and above line reverted his change).

Reply:
actually, will keep this change